### DISCUSSION TITLE
posts/mikrotik-hap-ax3-wifi/

### DISCUSSION THREAD

**Discussion by giscus** [DarrenOfficial/DarrenOfficial#11] [General]
# posts/mikrotik-hap-ax3-wifi/

The hAP AX3 [C53UiG5HPaxD2HPaxD] is an affordable Home Router Access Point, hence the name hAP. Although there's a lot of complaints in regards to wifi...

https://blog.darrennathanael.com/posts/mikrotik-hap-ax3-wifi/

**Comment by markkkky** [comment]
Thank you for the post Darren! 

I’ve had issues with the hAP ax3 too. 

Would you mind fleshing out your process for doing both the RouterBoard upgrade and the RouterOS?  I think I’ve only being doing the RouterOS by selecting System > Packages > Check for updates. I’m running 7.14.2 on the OS, but not sure how to upgrade the RouterBoard?

**Reply by DarrenOfficial** [reply]
![image](https://github.com/DarrenOfficial/DarrenOfficial/assets/55406094/d0e2729d-ba89-40db-a0e2-e146960b7f9c)
